packages/assistify-ai/server/SmartiProxy.js

```diff
@@ -33,15 +33,14 @@ export class SmartiProxy {
 	 * @returns {Object}
 	 */
 	static propagateToSmarti(method, path, body = null) {
-
-		const url = encodeURI(`${ SmartiProxy.smartiUrl }${ path }`);
+		const url = `${ SmartiProxy.smartiUrl }${ path }`;
```

Comment:
@Peym4n where'd you do the encoding of the URI? Should this not be done as close to the http-call as possible

Reply:
When we call `propagateToSmarti` and provide it with a path, the path should already be encoded. Currently the only call that requires encoding because of complex values is in the [`searchConversations` method](https://github.com/redlink-gmbh/Rocket.Chat/blob/80b7a913c86126629b3ece57c447e131feb51268/packages/assistify-ai/server/methods/SmartiWidgetBackend.js#L104) and I use node's `querystring` module for that.